recipe/meta.yaml

```diff
@@ -15,15 +15,27 @@ build:
 
 requirements:
     build:
+        - toolchain
         - gcc
         - perl
+        - gcc  # [osx]
+        - libgfortran  # [linux]
     run:
-        - gcc
+        - libgfortran  # [not win]
```

Comment:
Indeed you do need `libquadmath` (I just checked the error on Linux). However, I do believe that `gcc` is a run-time dependency here b/c users will need it do compile things later on. Like you are doing in the test.

At least we could do `- gcc  # [not win]` in the test below.